pkgs/tools/networking/termshark/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "termshark-${version}";
```

Comment:
If `name` is not set and `pname` is set, then set `name` to `${pname}-${version}`.

```suggestion
  pname = "termshark";
```